src/secrets/mod.rs

```diff
@@ -10,6 +10,36 @@ use crate::error::Result;
 
 /// Interface for secrets resolver.
 /// Resolves secrets such as private keys to be used for signing and encryption.
+#[cfg(feature = "uniffi")]
```

Comment:
cfg_attr?

Reply:
The same as in https://github.com/sicpa-dlab/didcomm-rust/pull/48#discussion_r756854133